build_tools/cmake/test.sh

```diff
@@ -62,4 +62,4 @@ fi
 label_exclude_regex="($(IFS="|" ; echo "${label_exclude_args[*]?}"))"
 
 cd ${ROOT_DIR?}/build
-ctest --label-exclude "${label_exclude_regex?}"
+ctest --output-on-failure --label-exclude "${label_exclude_regex?}"
```

Comment:
I've noticed in the past that output gets thoroughly scrambled by parallel tests. Does that happen here? There's an [ancient issue](https://github.com/google/iree/issues/865) to look into actually giving useful output, which I... kind of forgot about :see_no_evil:

`--output-on-failure` seems like an OK option in the meantime, even if it is scrambled

Reply:
Yup. I would like it to be more verbose if something goes wrong. I haven't hit the issue you mentioned regarding parallel build. We can revert the change later if it turns out to be annoying.